components/about-us/sub-pages/Governance/kickstart.tsx

```diff
@@ -20,10 +20,12 @@ export default function GovernanceKickstartComponent() {
         spacing={{ base: 4, md: 6 }}
         py={{ base: 10, md: 18 }}
         mt={'4rem'}
+        position='relative'
+        zIndex='docked'
```

Comment:
QUESTION : this is cool

Nice work taking advantage for the Chakra portals. Can you explain why you needed to this? Like what would happen if you didn't use this